### ISSUE TITLE
Bot doesnt Follow Up Issues

### ISSUE THREAD

**Issue by jordan-ae** [ubiquity/ubiquibot#926]
@0x4007 The flow to unassign a dormant contributor from a PR is to send in reminders or follow ups to make sure the collaborator is aware about the fact that the deadline to submit a PR or open a draft has been exceeded. Then if there's no response then the bot goes ahead and unassigns the collaborator. But right now from test on a local instance and a couple of instances I've seen in production the bot doesnt do any follow ups and goes ahead to unassign without warnings.

**Comment by 0x4007** [comment]
Yes this is a known issue that will be addressed in the next version. Thanks for reporting this but maintenance on the first version of the bot is halted unless it's a special circumstance (like something payment related).
